does not build for me (x86_64-linux -> aarch64-multiplatform):
```sh
$ git fetch upstream pull/373047/head:pr-373047
$ git switch pr-373047
$ nix-build -A pkgsCross.aarch64-multiplatform.opencolorio
this derivation will be built:
  /nix/store/20aim1vj0l2gln36hi50c15jrf54hvn3-opencolorio-aarch64-unknown-linux-gnu-2.4.1.drv
building '/nix/store/20aim1vj0l2gln36hi50c15jrf54hvn3-opencolorio-aarch64-unknown-linux-gnu-2.4.1.drv'...
Running phase: unpackPhase
unpacking source archive /nix/store/998ridqzcyfhd2mh28hadv88lqcc6mgc-source
source root is source
Running phase: patchPhase
applying patch /nix/store/jg1r5ifh2i3s6klys9q45ap3zf2qsncx-broken-cmake-zlib-version.patch
patching file src/cmake/Config.cmake.in
Hunk #1 succeeded at 46 (offset 3 lines).
applying patch /nix/store/lv968mbnljhz08v2pksh25j69lgj5m73-line-numbers.patch
patching file tests/cpu/fileformats/FileFormatCTF_tests.cpp
Hunk #1 succeeded at 1454 (offset 38 lines).
Running phase: updateAutotoolsGnuConfigScriptsPhase
Running phase: updateAutotoolsGnuConfigScriptsPhase
Running phase: configurePhase
fixing cmake files...
cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF -DCMAKE_INSTALL_LOCALEDIR=/nix/store/6i5xzwf7zd7g3dy82f103gffk2kxl5sg-opencolorio-aarch64-unknown-linux-gnu-2.4.1/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/6i5xzwf7zd7g3dy82f103gffk2kxl5sg-opencolorio-aarch64-unknown-linux-gnu-2.4.1/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/6i5xzwf7zd7g3dy82f103gffk2kxl5sg-opencolorio-aarch64-unknown-linux-gnu-2.4.1/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/6i5xzwf7zd7g3dy82f103gffk2kxl5sg-opencolorio-aarch64-unknown-linux-gnu-2.4.1/share/doc/OpenColorIO -DCMAKE_INSTALL_INFODIR=/nix/store/6i5xzwf7zd7g3dy82f103gffk2kxl5sg-opencolorio-aarch64-unknown-linux-gnu-2.4.1/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/6i5xzwf7zd7g3dy82f103gffk2kxl5sg-opencolorio-aarch64-unknown-linux-gnu-2.4.1/share/man -DCMAKE_INSTALL_OLDINCLUDEDIR=/nix/store/6i5xzwf7zd7g3dy82f103gffk2kxl5sg-opencolorio-aarch64-unknown-linux-gnu-2.4.1/include -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/6i5xzwf7zd7g3dy82f103gffk2kxl5sg-opencolorio-aarch64-unknown-linux-gnu-2.4.1/include -DCMAKE_INSTALL_SBINDIR=/nix/store/6i5xzwf7zd7g3dy82f103gffk2kxl5sg-opencolorio-aarch64-unknown-linux-gnu-2.4.1/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/6i5xzwf7zd7g3dy82f103gffk2kxl5sg-opencolorio-aarch64-unknown-linux-gnu-2.4.1/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/6i5xzwf7zd7g3dy82f103gffk2kxl5sg-opencolorio-aarch64-unknown-linux-gnu-2.4.1/lib -DCMAKE_POLICY_DEFAULT_CMP0025=NEW -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_STRIP=/nix/store/97hiadr3dgv6kj4w9mfx20ax09s8cyn3-aarch64-unknown-linux-gnu-gcc-wrapper-14-20241116/bin/aarch64-unknown-linux-gnu-strip -DCMAKE_RANLIB=/nix/store/97hiadr3dgv6kj4w9mfx20ax09s8cyn3-aarch64-unknown-linux-gnu-gcc-wrapper-14-20241116/bin/aarch64-unknown-linux-gnu-ranlib -DCMAKE_AR=/nix/store/97hiadr3dgv6kj4w9mfx20ax09s8cyn3-aarch64-unknown-linux-gnu-gcc-wrapper-14-20241116/bin/aarch64-unknown-linux-gnu-ar -DCMAKE_C_COMPILER=aarch64-unknown-linux-gnu-gcc -DCMAKE_CXX_COMPILER=aarch64-unknown-linux-gnu-g++ -DCMAKE_INSTALL_PREFIX=/nix/store/6i5xzwf7zd7g3dy82f103gffk2kxl5sg-opencolorio-aarch64-unknown-linux-gnu-2.4.1 -DOCIO_INSTALL_EXT_PACKAGES=NONE -DOCIO_USE_SSE2NEON=OFF -DOCIO_BUILD_GPU_TESTS=OFF -Dminizip-ng_INCLUDE_DIR=/nix/store/hqish8s3ph990kbmimp7vlv727adk847-minizip-ng-aarch64-unknown-linux-gnu-4.0.8/include/minizip-ng -DCMAKE_SYSTEM_NAME=Linux -DCMAKE_SYSTEM_PROCESSOR=aarch64 -DCMAKE_HOST_SYSTEM_NAME=Linux -DCMAKE_HOST_SYSTEM_PROCESSOR=x86_64 -DPYBIND11_PYTHONLIBS_OVERWRITE=OFF -DPYTHON_EXECUTABLE=/nix/store/c9m6yd8fg1flz2j5r4bif1ib5j20a0cy-python3-3.12.8/bin/python3.12 -DPYTHON_INCLUDE_DIR=/nix/store/sv1zkz9xrnzdm866qync4glny45vsqfb-python3-aarch64-unknown-linux-gnu-3.12.8/include/python3.12 -DPYTHON_SITE_PACKAGES=/nix/store/sv1zkz9xrnzdm866qync4glny45vsqfb-python3-aarch64-unknown-linux-gnu-3.12.8/lib/python3.12/site-packages
-- The CXX compiler identification is GNU 14.2.1
-- The C compiler identification is GNU 14.2.1
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /nix/store/97hiadr3dgv6kj4w9mfx20ax09s8cyn3-aarch64-unknown-linux-gnu-gcc-wrapper-14-20241116/bin/aarch64-unknown-linux-gnu-g++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /nix/store/97hiadr3dgv6kj4w9mfx20ax09s8cyn3-aarch64-unknown-linux-gnu-gcc-wrapper-14-20241116/bin/aarch64-unknown-linux-gnu-gcc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Setting C++ version to '14' as none was specified.
-- Performing Test COMPILER_SUPPORTS_CXX14
-- Performing Test COMPILER_SUPPORTS_CXX14 - Success
--
-- Checking for GPU configuration...
-- Found OpenGL (no version information)
-- Found GLEW (version "2.2.0")
-- Found GLUT (no version information)
-- GLVND supported
-- Performing Test COMPILER_SUPPORTS_ARM_NEON
-- Performing Test COMPILER_SUPPORTS_ARM_NEON - Success
-- Disabling SIMD optimizations, as the target doesn't support them
-- Setting SOVERSION to '2.4' as none was specified.
--
-- Missing a dependency? Try the following possibilities:
-- If the package provides CMake's configuration file, use -D<pkg>_DIR=<path to folder>.
-- If it doesn't provide it, try -D<pkg>_ROOT=<path to folder with lib and includes>.
-- Alternatively, try -D<pkg>_LIBRARY=<path to lib file> and/or -D<pkg>_INCLUDE_DIR=<path to folder>.
--
-- Please refer to the find module under share/cmake/modules for extra information.
--
-- Checking for mandatory dependencies...
-- Found expat (version "2.6.4")
-- Found yaml-cpp (version "0.8.0")
-- Found pystring (no version information)
-- Found Imath (version "3.1.12")
-- Found ZLIB (version "1.3.1")
-- Found minizip-ng (version "4.0.8")
--
-- Checking for optional dependencies...
-- Found lcms2 (version "2.16")
-- Could NOT find Python (no version specified)
CMake Error at share/cmake/macros/ocio_handle_dependency.cmake:220 (message):
  Python is required, will abort at the end.
Call Stack (most recent call first):
  share/cmake/modules/FindExtPackages.cmake:170 (ocio_handle_dependency)
  CMakeLists.txt:449 (include)
```

--- a/pkgs/development/libraries/opencolorio/default.nix
+++ b/pkgs/development/libraries/opencolorio/default.nix
@@ -8,6 +8,7 @@
   pystring,
   imath,
   minizip-ng,
+  zlib,
   # Only required on Linux
   glew,
   libglut,
@@ -51,7 +52,7 @@ stdenv.mkDerivation rec {
       --replace 'OCIO_ADD_TEST(Config, virtual_display_with_active_displays)' 'static void _skip_virtual_display_with_active_displays()'
   '';
 
-  nativeBuildInputs = [ cmake ];
+  nativeBuildInputs = [ cmake ] ++ lib.optionals pythonBindings [ python3Packages.python ];
   buildInputs =
     [
       expat
@@ -59,6 +60,7 @@ stdenv.mkDerivation rec {
       pystring
       imath
       minizip-ng
+      zlib
     ]
     ++ lib.optionals stdenv.hostPlatform.isLinux [
       glew

